### PR TITLE
Added IgnoreLatestTimeToReachSocDateOnWeekend

### PR DESCRIPTION
**Pull request by b1ggi**
Charging for "PV only" and "Spot Price + PV" only on weekdays.
Needs more testing, did a few tests

### CODE REVIEW

**Comment by b1ggi** [comment]
Should solve https://github.com/pkuehnel/TeslaSolarCharger/issues/846

File: TeslaSolarCharger.Model/Entities/TeslaSolarCharger/Car.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public class Car
     public DateTime LatestTimeToReachSoC { get; set; }
 
     public bool IgnoreLatestTimeToReachSocDate { get; set; }
+    public bool IgnoreLatestTimeToReachSocDateOnWeekdays { get; set; }
```
Comment (pkuehnel) [inline TeslaSolarCharger.Model/Entities/TeslaSolarCharger/Car.cs]: Shouldn't all these variables, migrations and methods be called IgnoreOnWeekends

Comment (b1ggi) [reply TeslaSolarCharger.Model/Entities/TeslaSolarCharger/Car.cs]: I Will change it

**Comment by pkuehnel** [comment]
Sorry for taking so long to integrate this but due to all the Tesla API changes I have so many other issues that I don't have the time to check this PR.

**Comment by b1ggi** [comment]
> Sorry for taking so long to integrate this but due to all the Tesla API changes I have so many other issues that I don't have the time to check this PR.

No problem, its not as important

**Pull request merged**